src/main/java/org/reactome/server/result/CustomPathway.java

```diff
@@ -0,0 +1,108 @@
+package org.reactome.server.result;
+
+
+import org.reactome.server.graph.domain.model.Person;
+
+import java.util.Collection;
+
+
+public class CustomPathway {
+    private String displayName;
+    private String doi;
+    private String stId;
+    private String species;
+    private String releaseDate;
+
+    private String reviseDate;
+    private String releaseStatus;
+
+
+    private Collection<Person> authors;
+    private Collection<Person> reviewers;
+    private Collection<Person> editors;
+
+    public Collection<Person> getAuthors() {
+        return authors;
+    }
+
+    public void setAuthors(Collection<Person> authors) {
+        this.authors = authors;
+    }
+
+    public Collection<Person> getReviewers() {
+        return reviewers;
+    }
+
+    public void setReviewers(Collection<Person> reviewers) {
+        this.reviewers = reviewers;
+    }
+
+    public Collection<Person> getEditors() {
+        return editors;
+    }
+
+    public void setEditors(Collection<Person> editors) {
+        this.editors = editors;
+    }
+
+    public String getReviseDate() {
+        return reviseDate;
+    }
+
+    public void setReviseDate(String reviseDate) {
+        this.reviseDate = reviseDate;
+    }
+
+    public CustomPathway() {
```

Comment:
Just being picky with the java convention here. A class should follow <package> <imports> <attributes> <constructors> <getters/setter>.

Reply:
ok, will redo it